networkx/algorithms/centrality/eigenvector.py

```diff
@@ -297,33 +305,35 @@ def eigenvector_centrality_numpy(G, weight=None, max_iter=50, tol=0):
     References
     ----------
     .. [1] Abraham Berman and Robert J. Plemmons.
-       "Nonnegative Matrices in the Mathematical Sciences."
-       Classics in Applied Mathematics. SIAM, 1994.
+        "Nonnegative Matrices in the Mathematical Sciences".
+        Classics in Applied Mathematics. SIAM, 1994.
 
     .. [2] Edmund Landau.
-       "Zur relativen Wertbemessung der Turnierresultate."
-       Deutsches Wochenschach, 11:366–369, 1895.
+        "Zur relativen Wertbemessung der Turnierresultate".
+        Deutsches Wochenschach, 11:366--369, 1895.
 
     .. [3] Teh-Hsing Wei.
-       "The Algebraic Foundations of Ranking Theory."
-       PhD thesis, University of Cambridge, 1952.
+        "The Algebraic Foundations of Ranking Theory".
+        PhD thesis, University of Cambridge, 1952.
 
     .. [4] Maurice G. Kendall.
-       "Further contributions to the theory of paired comparisons."
-       Biometrics, 11(1):43–62, 1955.
-       https://www.jstor.org/stable/3001479
+        "Further contributions to the theory of paired comparisons".
```

Comment:
FWIW the recommendation of the [numpydoc style guide](https://numpydoc.readthedocs.io/en/latest/format.html#references) is to keep multi-line text for references aligned with the opening bracket.

In practice this doesn't matter, it should all parse and render correctly. I mostly just mention it as I think minimizing the number of white-space-only changes in docstrings eases review! Just my two cents though, certainly not a blocker

Reply:
Oh, I didn't catch that when skimming the numpydoc style guide. Thanks for taking the time to explain!